frontend/src/components/SearchBar/ParentGroupedFilter.tsx

```diff
@@ -29,7 +31,10 @@ export const ParentGroupedFilter: React.FC<IParentGroupedFilterProps> = ({
       bsSize={inputSize}
       filterBy={filterBy}
       placeholder={placeholder}
-      filter={true}
+      onChange={vals => {
```

Comment:
This overrides to display the name?  Does it still use `id` value for querying?

Reply:
Yes still uses the id for querying, it alters the filter value onFilterChange in MapFilterBar.tsx

Reply:
Can you confirm this is working on the GRETransformForm? this appears to be the only other place the typeahead is used for agencies.